src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -927,6 +927,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmission_OverwriteNotice_Antigen_Text" = "Sie haben bereits einen Schnelltest registriert. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest registrieren, wird der erste Schnelltest aus der App gelöscht.";
 
+/* ExposureSubmission AntigenTest Profile */
+
+"ExposureSubmission_Antigen_Profile_Header_Text" = "Schnelltest-Profil";
+
+"ExposureSubmission_Antigen_Profile_QRCode_Image_Description" = "Ein QR Code mit Ihrem Schnelltest-Profils";
```

Comment:
Fo consistency I'd write QR Code like this: "QR-Code"

Reply:
@janetback Could you confirm this change?